### ISSUE TITLE
it doesn't work for me 

### ISSUE THREAD

**Issue by zzzgit** [jest-community/eslint-plugin-jest#245]
```yml
env:
  node: true
  commonjs: true
  es6: true
  jest: true
plugin:
  - 'jest'
rules:
  jest/no-identical-title: 
    - error

```

I don't know why it doesn't work.
I have installed `eslint` and `eslint-plugin-jest` also.

**Comment by SimenB** [comment]
Can you put together s reproduction we can pull down and test?

**Comment by zzzgit** [comment]
https://github.com/zzzgit/sample

Please check test.ts

**Comment by SimenB** [comment]
I had to fix the eslintrc file (`plugin` -> `plugins`)

```sh-session
$ npx eslint test/test.ts
Error: ESLint configuration in /Users/simen/sample/test/.eslintrc.yml is invalid:
        - Unexpected top-level property "plugin".
```

After that it worked fine

```sh-session
$ npx eslint test/test.ts

/Users/simen/sample/test/test.ts
  5:4  error  Extra semicolon                                               semi
  6:2  error  Test title is used multiple times in the same describe block  jest/no-identical-title
  8:4  error  Extra semicolon                                               semi

✖ 3 problems (3 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

What command are you running?

**Comment by zzzgit** [comment]
I am using a eslint-vs-plugin, instead of manually running a command.
The problem is that I haven't enable eslint support for .ts files in vs-code.
